app/controllers/passwords_controller.rb

```diff
@@ -43,6 +43,7 @@ def record_reset_page_loaded
   end
 
   def record_password_reset_failure(user)
-    EventLog.record_event(user, EventLog::PASSPHRASE_RESET_FAILURE)
+    message = "(errors: #{user.errors.full_messages.join(', ')})"
```

Comment:
Tried that, but it looked weird:

`join`:

> Errors: Passphrase can't be blank, Passphrase not strong enough. Consider adding a number, symbols or more letters to make it stronger.

vs

`to_sentence`:

> Errors: Passphrase can't be blank and Passphrase not strong enough. Consider adding a number, symbols or more letters to make it stronger.

In the second case it looks like the errors belong together.